ObjCVoiceQuickstart/ViewController.m

```diff
@@ -186,6 +204,9 @@ - (void)incomingCallCancelled:(TVOIncomingCall *)incomingCall {
         NSLog(@"Incoming (but not current) call from \"%@\" cancelled. Just ignore it.", incomingCall.from);
         return;
     }
+    
+    [self stopIncomingRingtone:nil];
+    [self playDisconnectSound];
```

Comment:
There is a way we can play the disconnect sound when rejecting/ignoring - via the `call:didDisconnect:` callback, but I need to figure out the correct order of methods so that the sound playback doesn't mess with the pj-media thingy when calling reject. Currently `reject` method will trigger an INVITE event in PJSIP, which will somewhere configure the AVAudioSession again...